### PR TITLE
feat(ui): add Spanish localization support

### PR DESCRIPTION
**Pull request by bazookon**
## Description

*Replace this paragraph with a description of what this PR is doing. If you're modifying existing behavior, describe the existing behavior, how this PR is changing it, and what motivated the change.*

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/flutter/flutter/issues). Indicate, which of these issues are resolved or fixed by this PR. Note that you'll have to prefix the issue numbers with flutter/flutter#.*

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [ ] All existing and new tests are passing.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`melos run analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [ ] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/FirebaseExtended/flutterfire/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+  @override
+  final String timestampLabel;
+  @override
+  final String latitudeLabel;
+  @override
+  final String longitudeLabel;
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
```
Comment (lesnitsky) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:169]:
```suggestion
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+    this.updateLabel = 'update',
+    this.northInitialLabel = 'N',
+    this.southInitialLabel = 'S',
+    this.westInitialLabel = 'W',
+    this.eastInitialLabel = 'E',
+    this.timestampLabel = 'timestamp',
+    this.longitudeLabel = 'longitude',
+    this.latitudeLabel = 'latitude',
+    this.geopointLabel = 'geopoint',
+    this.referenceLabel = 'reference',
```
Comment (lesnitsky) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:262]:
```suggestion
```

**Review by lesnitsky — CHANGES_REQUESTED** [review]
~I see labels that do not exist in a base `FlutterFireUILocalizationLabels ` class, please drop those~

Ignore me, I forgot these are `DataTable` labels

**Review by lesnitsky — APPROVED** [review]

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+    this.nullLabel = 'null',
+    this.numberLabel = 'number',
+    this.stringLabel = 'string',
+    this.typeLabel = 'type',
+    this.valueLabel = 'value',
+    this.cancelLabel = 'cancel',
+    this.updateLabel = 'update',
+    this.northInitialLabel = 'N',
+    this.southInitialLabel = 'S',
+    this.westInitialLabel = 'W',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:256]:
```suggestion
    this.westInitialLabel = 'O',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+  @override
+  final String longitudeLabel;
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
+
+
+  const EsLocalizations({
+    this.emailInputLabel = 'Correo Electrónico',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:173]: I think this is usually lowercase:
```suggestion
    this.emailInputLabel = 'Correo electrónico',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+    this.passwordInputLabel = 'Contraseña',
+    this.signInActionText = 'Ingresar',
+    this.registerActionText = 'Registrarse',
+    this.signInButtonText = 'Ingresar',
+    this.registerButtonText = 'Registrarse',
+    this.linkEmailButtonText = 'Siguiente',
+    this.signInWithPhoneButtonText = 'Entra con phone',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:180]:
```suggestion
    this.signInWithPhoneButtonText = 'Entra con teléfono',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+  @override
+  final String referenceLabel;
+
+
+  const EsLocalizations({
+    this.emailInputLabel = 'Correo Electrónico',
+    this.passwordInputLabel = 'Contraseña',
+    this.signInActionText = 'Ingresar',
+    this.registerActionText = 'Registrarse',
+    this.signInButtonText = 'Ingresar',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:177]: Not sure where this is usually used, but sometimes it's also shown as:
```suggestion
    this.signInButtonText = 'Iniciar sesión',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+    this.registerButtonText = 'Registrarse',
+    this.linkEmailButtonText = 'Siguiente',
+    this.signInWithPhoneButtonText = 'Entra con phone',
+    this.signInWithGoogleButtonText = 'Entra con Google',
+    this.signInWithAppleButtonText = 'Entra con Apple',
+    this.signInWithTwitterButtonText = 'Entra con Twitter',
+    this.signInWithFacebookButtonText = 'Entra con Facebook',
+    this.phoneVerificationViewTitleText = 'Ingresa tu número de telefono',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:185]:
```suggestion
    this.phoneVerificationViewTitleText = 'Ingresa tu número de teléfono',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 8]
```diff
@@ -0,0 +1,264 @@
...
+    this.verifyPhoneNumberButtonText = 'Siguiente',
+    this.verifyCodeButtonText = 'Verificar',
+    this.verifyingPhoneNumberViewTitle = 'Ingrese el código de SMS',
+    this.unknownError = 'Ocurrió un error desconocido',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:189]:
```suggestion
    this.unknownError = 'Ha ocurrido un error desconocido',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 9]
```diff
@@ -0,0 +1,264 @@
...
+    this.smsAutoresolutionFailedError =
+        'No se pudo resolver el código SMS automáticamente. Por favor ingrese su código manualmente',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:191]:
```suggestion
        'No se ha podido detectar el código SMS automáticamente. Por favor, ingrese su código manualmente',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 10]
```diff
@@ -0,0 +1,264 @@
...
+    this.smsCodeSentText = 'Código SMS enviado',
+    this.sendingSMSCodeText = 'Enviando código SMS ...',
+    this.verifyingSMSCodeText = 'Verificando el código SMS ...',
+    this.enterSMSCodeText = 'Ingrese el código SMS',
+    this.emailIsRequiredErrorText = 'El correo electrónico es requirido',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:196]:
```suggestion
    this.emailIsRequiredErrorText = 'El correo electrónico es obligatorio',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 11]
```diff
@@ -0,0 +1,264 @@
...
+    this.isNotAValidEmailErrorText = 'Ingresa un Correo Electrónico válido',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:197]:
```suggestion
    this.isNotAValidEmailErrorText = 'Ingresa un correo electrónico válido',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 12]
```diff
@@ -0,0 +1,264 @@
...
+    this.wrongOrNoPasswordErrorText =
+        'La contraseña no es válida o el usuario no tiene contraseña.',
+    this.signInText = 'Ingresar',
+    this.registerText = 'Registrarse',
+    this.registerHintText = '¿No tienes una cuenta?',
+    this.signInHintText = '¿Ya tienes una cuenta?',
+    this.signOutButtonText = 'Salir',
+    this.phoneInputLabel = 'Número de teléfono',
+    this.phoneNumberInvalidErrorText = 'El número de teléfono no es válido',
+    this.phoneNumberIsRequiredErrorText = 'El número de teléfono es requerido',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:211]:
```suggestion
    this.phoneNumberIsRequiredErrorText = 'El número de teléfono es obligatorio',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 13]
```diff
@@ -0,0 +1,264 @@
...
+    this.profile = 'Perfil',
+    this.name = 'Nombre',
+    this.deleteAccount = 'Eliminar cuenta',
+    this.passwordIsRequiredErrorText = 'La contraseña es requerida',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:215]:
```suggestion
    this.passwordIsRequiredErrorText = 'La contraseña es obligatoria',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 14]
```diff
@@ -0,0 +1,264 @@
...
+    this.confirmPasswordIsRequiredErrorText = 'Confirma tu contraseña',
+    this.confirmPasswordDoesNotMatchErrorText = 'Las contraseñas no coinciden',
+    this.confirmPasswordInputLabel = 'Confirma la contraseña',
+    this.forgotPasswordButtonLabel = '¿Olvidaste tu contraseña?',
+    this.forgotPasswordViewTitle = 'Olvidar contraseña',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:220]:
```suggestion
    this.forgotPasswordViewTitle = 'Contraseña olvidada',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 15]
```diff
@@ -0,0 +1,264 @@
...
+    this.resetPasswordButtonLabel = 'Restablecer contraseña',
+    this.verifyItsYouText = 'Verifica que eres tú',
+    this.differentMethodsSignInTitleText =
+        'Utilice uno de los siguientes métodos para iniciar sesión',
+    this.findProviderForEmailTitleText = 'Ingrese su correo electrónico para continuar',
+    this.continueText = 'Continuar',
+    this.countryCode = 'Código de país',
+    this.codeRequiredErrorText = 'El código de país es requerido',
```
Comment (Zazo032) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:228]:
```suggestion
    this.codeRequiredErrorText = 'El código de país es obligatorio',
```

**Review by Zazo032 — COMMENTED** [review]
¡Hola! I might have missed something, and some changes are style preference, but I suggested some changes to some texts. If the labels at the bottom are not meant for translation, they probably should be moved to another file

**Comment by Salakar** [comment]
Thanks for this @bazookon - would you mind taking a look at the review suggestions helpfully provided from @Zazo032

**Comment by eaceto** [comment]
Hi! There are a few missing accents, like in "teléfono", "código", "número". May I fork this and send a PR with the fixes?

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 16]
```diff
@@ -0,0 +1,264 @@
...
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
+
+
+  const EsLocalizations({
+    this.emailInputLabel = 'Correo Electrónico',
+    this.passwordInputLabel = 'Contraseña',
+    this.signInActionText = 'Ingresar',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:175]:
```suggestion
    this.signInActionText = 'Iniciar sesión',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart (hunk 6 above)
Comment (jamesblasco) [reply packages/flutterfire_ui/lib/src/i10n/lang/es.dart:177]: Yeah your suggestion is how it is usually displayed. Check for example Google in Spanish

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart (hunk 7 above)
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:185]:
```suggestion
    this.phoneVerificationViewTitleText = 'Introduce tu número de telefono',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 17]
```diff
@@ -0,0 +1,264 @@
...
+    this.linkEmailButtonText = 'Siguiente',
+    this.signInWithPhoneButtonText = 'Entra con phone',
+    this.signInWithGoogleButtonText = 'Entra con Google',
+    this.signInWithAppleButtonText = 'Entra con Apple',
+    this.signInWithTwitterButtonText = 'Entra con Twitter',
+    this.signInWithFacebookButtonText = 'Entra con Facebook',
+    this.phoneVerificationViewTitleText = 'Ingresa tu número de telefono',
+    this.verifyPhoneNumberButtonText = 'Siguiente',
+    this.verifyCodeButtonText = 'Verificar',
+    this.verifyingPhoneNumberViewTitle = 'Ingrese el código de SMS',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:188]:
```suggestion
    this.verifyingPhoneNumberViewTitle = 'Introduce el código del SMS',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart (hunk 5 above)
Comment (jamesblasco) [reply packages/flutterfire_ui/lib/src/i10n/lang/es.dart:180]:
```suggestion
    this.signInWithPhoneButtonText = 'Entra con tu teléfono',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 18]
```diff
@@ -0,0 +1,264 @@
...
+    this.unknownError = 'Ocurrió un error desconocido',
+    this.smsAutoresolutionFailedError =
+        'No se pudo resolver el código SMS automáticamente. Por favor ingrese su código manualmente',
+    this.smsCodeSentText = 'Código SMS enviado',
+    this.sendingSMSCodeText = 'Enviando código SMS ...',
+    this.verifyingSMSCodeText = 'Verificando el código SMS ...',
+    this.enterSMSCodeText = 'Ingrese el código SMS',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:195]:
```suggestion
    this.enterSMSCodeText = 'Introduce el código SMS',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart (hunk 11 above)
Comment (jamesblasco) [reply packages/flutterfire_ui/lib/src/i10n/lang/es.dart:197]:
```suggestion
    this.isNotAValidEmailErrorText = 'Introduce un correo electrónico válido',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 19]
```diff
@@ -0,0 +1,264 @@
...
+    this.emailIsRequiredErrorText = 'El correo electrónico es requirido',
+    this.isNotAValidEmailErrorText = 'Ingresa un Correo Electrónico válido',
+    this.userNotFoundErrorText = 'La cuenta NO existe',
+    this.emailTakenErrorText = 'La cuenta con ese correo electrónico ya existe',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:199]:
```suggestion
    this.emailTakenErrorText = 'Ya existe una cuenta con este correo electrónico',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 20]
```diff
@@ -0,0 +1,264 @@
...
+    this.unknownError = 'Ocurrió un error desconocido',
+    this.smsAutoresolutionFailedError =
+        'No se pudo resolver el código SMS automáticamente. Por favor ingrese su código manualmente',
+    this.smsCodeSentText = 'Código SMS enviado',
+    this.sendingSMSCodeText = 'Enviando código SMS ...',
+    this.verifyingSMSCodeText = 'Verificando el código SMS ...',
+    this.enterSMSCodeText = 'Ingrese el código SMS',
+    this.emailIsRequiredErrorText = 'El correo electrónico es requirido',
+    this.isNotAValidEmailErrorText = 'Ingresa un Correo Electrónico válido',
+    this.userNotFoundErrorText = 'La cuenta NO existe',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:198]:
```suggestion
    this.userNotFoundErrorText = 'No existe una cuenta con este usuario',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 21]
```diff
@@ -0,0 +1,264 @@
...
+    this.emailTakenErrorText = 'La cuenta con ese correo electrónico ya existe',
+    this.accessDisabledErrorText =
+        'El acceso a esta cuenta se ha inhabilitado temporalmente',
+    this.wrongOrNoPasswordErrorText =
+        'La contraseña no es válida o el usuario no tiene contraseña.',
+    this.signInText = 'Ingresar',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:204]:
```suggestion
    this.signInText = 'Iniciar sesión',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 22]
```diff
@@ -0,0 +1,264 @@
...
+    this.registerText = 'Registrarse',
+    this.registerHintText = '¿No tienes una cuenta?',
+    this.signInHintText = '¿Ya tienes una cuenta?',
+    this.signOutButtonText = 'Salir',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:208]:
```suggestion
    this.signOutButtonText = 'Cerrar sesión',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 23]
```diff
@@ -0,0 +1,264 @@
...
+    this.phoneNumberInvalidErrorText = 'El número de teléfono no es válido',
+    this.phoneNumberIsRequiredErrorText = 'El número de teléfono es requerido',
+    this.profile = 'Perfil',
+    this.name = 'Nombre',
+    this.deleteAccount = 'Eliminar cuenta',
+    this.passwordIsRequiredErrorText = 'La contraseña es requerida',
+    this.confirmPasswordIsRequiredErrorText = 'Confirma tu contraseña',
+    this.confirmPasswordDoesNotMatchErrorText = 'Las contraseñas no coinciden',
+    this.confirmPasswordInputLabel = 'Confirma la contraseña',
+    this.forgotPasswordButtonLabel = '¿Olvidaste tu contraseña?',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:219]:
```suggestion
    this.forgotPasswordButtonLabel = '¿Has olvidado tu contraseña?',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 24]
```diff
@@ -0,0 +1,264 @@
...
+    this.forgotPasswordViewTitle = 'Olvidar contraseña',
+    this.resetPasswordButtonLabel = 'Restablecer contraseña',
+    this.verifyItsYouText = 'Verifica que eres tú',
+    this.differentMethodsSignInTitleText =
+        'Utilice uno de los siguientes métodos para iniciar sesión',
+    this.findProviderForEmailTitleText = 'Ingrese su correo electrónico para continuar',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:225]:
```suggestion
    this.findProviderForEmailTitleText = 'Introduce su correo electrónico para continuar',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 25]
```diff
@@ -0,0 +1,264 @@
...
+    this.continueText = 'Continuar',
+    this.countryCode = 'Código de país',
+    this.codeRequiredErrorText = 'El código de país es requerido',
+    this.invalidCountryCode = 'Código de país inválido',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:229]:
```suggestion
    this.invalidCountryCode = 'El código del país es inválido',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 26]
```diff
@@ -0,0 +1,264 @@
...
+    this.provideEmail = 'Proporcione su correo electrónico y contraseña',
+    this.goBackButtonLabel = 'Volver',
+    this.passwordResetEmailSentText =
+        'Le enviamos un correo electrónico con un enlace para restablecer su contraseña. Por favor revise su correo electrónico.',
+    this.forgotPasswordHintText =
+        'Proporcione su correo electrónico y le enviaremos un enlace para restablecer su contraseña',
+    this.emailLinkSignInButtonLabel = 'Iniciar sesión con enlace mágico',
+    this.signInWithEmailLinkViewTitleText = 'Iniciar sesión con enlace mágico',
+    this.signInWithEmailLinkSentText =
+        'Le enviamos un correo electrónico con un enlace mágico. Revise su correo electrónico y siga el enlace para iniciar sesión',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:242]:
```suggestion
        'Le hemos enviado un correo electrónico con un enlace mágico. Revise su correo electrónico y siga el enlace para iniciar sesión',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 27]
```diff
@@ -0,0 +1,264 @@
...
+        'Utilice uno de los siguientes métodos para iniciar sesión',
+    this.findProviderForEmailTitleText = 'Ingrese su correo electrónico para continuar',
+    this.continueText = 'Continuar',
+    this.countryCode = 'Código de país',
+    this.codeRequiredErrorText = 'El código de país es requerido',
+    this.invalidCountryCode = 'Código de país inválido',
+    this.chooseACountry = 'Seleccione un país',
+    this.enableMoreSignInMethods = 'Habilitar más métodos de inicio de sesión',
+    this.signInMethods = 'Métodos de inicio de sesión',
+    this.provideEmail = 'Proporcione su correo electrónico y contraseña',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:233]:
```suggestion
    this.provideEmail = 'Introduce su correo electrónico y contraseña',
```

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart [hunk 28]
```diff
@@ -0,0 +1,264 @@
...
+    this.goBackButtonLabel = 'Volver',
+    this.passwordResetEmailSentText =
+        'Le enviamos un correo electrónico con un enlace para restablecer su contraseña. Por favor revise su correo electrónico.',
+    this.forgotPasswordHintText =
+        'Proporcione su correo electrónico y le enviaremos un enlace para restablecer su contraseña',
```
Comment (jamesblasco) [inline packages/flutterfire_ui/lib/src/i10n/lang/es.dart:238]:
```suggestion
        'Introduce su correo electrónico y le enviaremos un enlace para restablecer su contraseña',
```

**Review by jamesblasco — CHANGES_REQUESTED** [review]

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart (hunk 28 above)
Comment (eaceto) [reply packages/flutterfire_ui/lib/src/i10n/lang/es.dart:238]: I feel there is a mix of formal and informal language here. 

"Introduce" is informal, where "su" is formal.

It should be "introduce tu" (informal) or "introduzca su" (formal)

**Comment by Salakar** [comment]
> Hi! There are a few missing accents, like in "teléfono", "código", "número". May I fork this and send a PR with the fixes?

@eaceto would you mind adding github review suggestions instead on this PR? Thanks

File: packages/flutterfire_ui/lib/src/i10n/lang/es.dart (hunk 28 above)
Comment (jamesblasco) [reply packages/flutterfire_ui/lib/src/i10n/lang/es.dart]: Introduce is the same for 'tu' and 'usted'. Introduzca is the subjunctive form of the verb

**Review by bazookon — COMMENTED** [review]
Fix some issues with formality

**Review by bazookon — COMMENTED** [review]
Fix some issues with formality

**Comment by Salakar** [comment]
@bazookon, @eaceto, @jamesblasco - thanks for contributions so far

Would either of you mind looking at these items that are untranslated at the bottom of the file:

https://github.com/FirebaseExtended/flutterfire/pull/7716/files#diff-27d49876363ee03af36e58b77854f869b382ad3144867c10b811a362f69e290fR247-R265 : 
image


(and also a final review pass please 💙 )

**Review by bazookon — COMMENTED** [review]
@Salakar what do you think about this, we are done?

**Review by Salakar — APPROVED** [review]

**Pull request merged**